packages/admin/cms-admin/src/redirects/RedirectsTable.tsx

```diff
@@ -80,7 +80,7 @@ export function RedirectsTable({ linkBlock, scope }: Props): JSX.Element {
             field: "source",
             headerName: intl.formatMessage({ id: "comet.pages.redirects.redirect.source", defaultMessage: "Source" }),
             sortable: true,
-            flex: 1,
+            minWidth: 600,
```

Comment:
We also want to support mobile screens in the future. So this `minWidth` is too big. Maybe go down to 300 or 400